classes/ClaytonTestClass2.cls

```diff
@@ -0,0 +1,1215 @@
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method148(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method149(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method150(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method151(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method152(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method153(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method154(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method155(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method156(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method157(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method158(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method159(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method160(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method161(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method162(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method163(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method164(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method165(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method166(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method167(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method168(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method169(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method170(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method171(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method172(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method173(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method174(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method175(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method176(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method177(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method178(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method179(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method180(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method181(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method182(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method183(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method184(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method185(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method186(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method187(){
+		Integer i = 0;
```

Comment:
This variable doesn't seem to have a very meaningful name. I suggest to give this variable a concise but meaningful name that suggest its purpose.